Sources/WebDriver/Element.swift

```diff
@@ -98,6 +98,16 @@ public struct Element {
         try session.findElements(startingAt: self, locator: locator, waitTimeout: waitTimeout)
     }
 
+    /// Finds an element using a given locator, starting from this element, and throwing upon failure.
+    /// - Parameter locator: The locator strategy to use.
+    /// - Parameter description: A human-readable description of the element, included in thrown errors.
+    /// - Parameter waitTimeout: The amount of time to wait for element existence. Overrides the implicit wait timeout.
+    /// - Returns: The element that was found.
+    @discardableResult // for use as an assertion
+    public func requireElement(locator: ElementLocator, description: String? = nil, waitTimeout: TimeInterval? = nil) throws -> Element {
```

Comment:
When waitTimeout is not specified, doesn't requireElement wait for the same period as findElement? In the PR description, does 'best-case' implicit wait imply that a waitTimeout period must be provided here?

Reply:
"best case" means that we will exit early if the element is found before the timeout expires. The semantics are the same as for `findElements` but the API makes it clearer that this is meant to be used for positive expectations.